examples/vae/ss_vae_M2.py

```diff
@@ -405,13 +396,14 @@ def main(args):
 
     parser.add_argument('--cuda', action='store_true',
                         help="use GPU(s) to speed up training")
-    parser.add_argument('-ne', '--num-epochs', default=100, type=int,
+    parser.add_argument('-ne', '--num-epochs', default=50, type=int,
```

Comment:
can you change this back to what it was in dev (i think `-n`) to match the other examples